sdk/servicebus/Azure.Messaging.ServiceBus/stress/src/Roles/SessionSender.cs

```diff
@@ -0,0 +1,156 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Linq;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Messaging.ServiceBus;
+
+namespace Azure.Messaging.ServiceBus.Stress;
+
+/// <summary>
+///   The role responsible for running a <see cref="ServiceBusSender" \> with sessions, and testing its performance over
+///   a long period of time. It collects metrics about the run and sends them to application insights using a
+///   <see cref="TelemetryClient" \>. The metrics collected are garbage collection information, any exceptions
+///   thrown or heard, and how many messages are processed and read. It stops sending messages and cleans up resources
+///   at the end of the test run.
+/// </summary>
+///
+internal class SessionSender
```

Comment:
The SessionSender/Sender scenarios seem similar enough that it may be worth trying to reuse the code rather than duplicating. Whether by introducing a base class or just helper methods.